WPMedia/fusion-news-theme-blocks
# Pen 1540 ad section path transform

## Description
- remove section id transformation in ad block

## Jira Ticket
- [PEN-1540](https://arcpublishing.atlassian.net/browse/PEN-1540)

## Acceptance Criteria
- Clients could set their ads to serve on the section name exactly as it is in site service / url, so /daily-chronicle or lake-county-journal

## Test Steps
- pull this branch down locally, run `rm -rf ./node_modules && npx lerna clean && npm i && npx lerna bootstrap && cd blocks/header-nav-chain-block && npm i && cd ../..`
- in `Fusion-News-Theme`, run `npx fusion start -f -l  @wpmedia/ads-block@canary`
- once fusion started, visit `http://localhost/footer-titles/2020/12/03/test-story-for-ad-path//?_website=the-sun`
- open your developer tool and run `javascript: googletag.openConsole()` in the browser's console.
- confirm the ad unit's id does not transform the "-" to "_"
<img width="745" alt="Screen Shot 2020-12-03 at 8 09 36 PM" src="https://user-images.githubusercontent.com/4411687/101122027-62997700-35a6-11eb-8169-655530224dce.png">


## Effect Of Changes
### Before
<img width="657" alt="Screen Shot 2020-12-03 at 8 30 14 PM" src="https://user-images.githubusercontent.com/4411687/101122080-7fce4580-35a6-11eb-8a7a-a0d2adbab3a0.png">


### After
<img width="745" alt="Screen Shot 2020-12-03 at 8 09 36 PM" src="https://user-images.githubusercontent.com/4411687/101122037-69c08500-35a6-11eb-8d50-3dd74d34309f.png">


## Dependencies or Side Effects
_Examples of dependencies or side effects are:_
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Update to css framework or SDK
- Dependency on another PR that needs to be merged first

## Review Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).
